client/views/admin/users/UsersPage.js

```diff
@@ -110,7 +110,7 @@ function UsersPage() {
 						<VerticalBar.Close onClick={handleVerticalBarCloseButtonClick} />
 					</VerticalBar.Header>
 
-					{context === 'info' && <UserInfoWithData uid={id} onReload={reload} />}
+					{context === 'info' && id !== undefined && <UserInfoWithData uid={id} onReload={reload} />}
```

Comment:
```suggestion
					{context === 'info' && id && <UserInfoWithData uid={id} onReload={reload} />}
```

Reply:
In Javascript, we don't we to explicity check for undefined, we can rather chain it like this

Reply:
I have done the suggested changes. @murtaza98  please Review it

Reply:
Awesome... Thanks!!